classes/item.rb

```diff
@@ -1,19 +1,21 @@
-class Item
-  attr_accessor :publish_date, :genre, :author, :source, :label
-
-  def initialize(_id, publish_date, archived: false)
-    @id = Random.rand(1..1000)
-    @publish_date = publish_date
-    @archived = archived
-  end
-
-  def move_to_archivd
-    @archived = true if can_be_archived?
-  end
-
-  private
-
-  def can_be_archived?
-    return true if Time.new.year - published_date > 10
-  end
-end
+class Item
+  attr_accessor :publish_date, :genre, :author, :source, :label, :archived
+  attr_reader :id
+
+  def initialize(_id, publish_date, archived, genre)
+    @id = Random.rand(1..1000)
+    @publish_date = publish_date
+    @archived = archived
+    @genre = genre
+  end
+
+  def move_to_archive
+    @archived = true if can_be_archived?
+  end
+
+  private
+
+  def can_be_archived?
+    return true if Time.new.year - published_date > 10

```

Comment:
![165637503-4b332f67-10d6-4235-8c42-de784c21372c](https://user-images.githubusercontent.com/84803900/165666205-a8aaafd2-1e99-4829-8b74-f52cf05e2535.png)

Reply:
Please Deal with date like this